src/components/FilteredList/styles/FilteredList.css.js

```diff
@@ -0,0 +1,55 @@
+import styled from '../../styled'
+import baseStyles from '../../../styles/resets/baseStyles.css.js'
+import { getColor } from '../../../styles/utilities/color'
+import { setFontSize } from '../../../styles/utilities/font'
+
+import Badge from '../../Badge'
+
+export const ItemUI = styled('li')`
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  align-items: center;
+  color: ${getColor('charcoal.400')};
+`
+
+export const BadgeUI = styled(Badge)`
+  &.is-square {
+    background-color: ${getColor('grey.300')};
+    box-shadow: 0 0 0 1px ${getColor('grey.500')} inset;
+    margin-left: 5px;
+  }
```

Comment:
What do you think of moving this option to the `Badge` component? We need to use this badge variant in other places.

Reply:
👍